crates/compiler/test_mono/src/tests.rs

```diff
@@ -2271,14 +2334,17 @@ fn issue_4705() {
 
 #[mono_test(mode = "test", large_stack = "true")]
 fn issue_4749() {
-    indoc!(
+    &formatdoc!(
         r#"
-        interface Test exposes [] imports [TotallyNotJson]
+        interface Test exposes [] imports []
+
+        {ERR_DECODER_FMT}
 
         expect
             input = [82, 111, 99]
-            got = Decode.fromBytes input TotallyNotJson.json
+            got = Decode.fromBytes input (@ErrDecoder {{}})
             got == Ok "Roc"
+
         "#
     )
 }
```

Comment:
I don't understand issue #4749 enough to tell whether the changed code is testing the same thing as the previous.

Reply:
I'm not sure either -- I suspect it isn't relevant any longer. @ayazhafiz may be able to look at this. 

I guess we could replace it with the repro Ayaz mentioned in the issue?

```roc
expect
    got : [Y[Y1, Y2], Z [Z1, Z2]]
    got = Z Z1

    t : [A [A1, A2]]
    t = A A1

    got == t
```

Reply:
I tried that, and it doesn't compile. I think the type checker has maybe gotten smarter since this ticket was opened?

Reply:
This is working for me, I'm guessing it's an open/closed record thing

```roc
module []

expect
    got : [Y [Y1, Y2], Z [Z1, Z2]]_
    got = Z Z1

    t : [A [A1, A2]]_
    t = A A1

    got != t

```

Reply:
ah, thanks, I would not have thought to try that